lib/aggregation/aggregator/src/index.js

```diff
@@ -410,12 +410,17 @@ const aggregate = function *(aggrs, u) {
   const newa = a ?
     extend(reviveOrg(JSON.parse(JSON.stringify(a))), {
       account_id: u.account_id,
-      end: u.end
+      start: u.start,
+      end: u.end,
+      resource_instance_id: u.resource_instance_id,
+      consumer_id: u.consumer_id
     }) :
     extend(newOrg(u.organization_id), {
+      account_id: u.account_id,
       start: u.start,
       end: u.end,
-      account_id: u.account_id
+      resource_instance_id: u.resource_instance_id,
```

Comment:
Shouldn't the ```start```, ```end``` and ```resource_instance_id``` fields be added to the consumer level aggregated usage doc as well? If the intent is to help a downstream app do dup detection, then AIUI an app interested in consumer level aggregation would need these fields too in that doc, right?